glpk/meta.yaml

```diff
@@ -17,6 +17,16 @@ requirements:
   build:
     - gmp      [not win]
 
+test:
```

Comment:
Thanks I'm glad you like it!

I tried adding the following block:
```
test:
  requires:
    - cython
```

Any idea why I'm getting this error message?
```
BUILD END: glpk-4.57-vc14_0
TEST START: glpk-4.57-vc14_0
Fetching package metadata: ......
Solving package specifications: ..
Error: Unsatisfiable package specifications.
Generating hint:
[      COMPLETE      ]|##################################################| 100%

Hint: 'glpk' has unsatisfiable dependencies (see 'conda info glpk')
```